lib/ChangeDueDateDialog/LoanList.js

```diff
@@ -66,6 +69,12 @@ const LoanList = (props) => {
 
   const rowUpdater = (rowData) => requestCounts[rowData.itemId];
 
+  const setCheckboxLabel = (id) => {
+    const currentLoan = loans.find(loan => loan.id === id).item;
+
+    return `checkbox-${kebabCase(currentLoan.title)}-${currentLoan.barcode}`;
```

Comment:
Since this will be read aloud by a screen reader, we shouldn't hard-code "checkbox" in there. @JohnC-80, is there any particular guidance about including an `input` element's type in its `aria-label` value? It seems to me that either we can omit "checkbox" (and likely the dashes, and kebabCase too; this is a label to be read aloud to a human; not an identifier to be used for a computer), or we at least need to pull it through react-intl and `formatMessage()`.

Reply:
Hi @zburke Thank you for the review! Updated the PR. Could you take a look, please?